packages/web/src/components/add-to-collection/desktop/AddToCollectionModal.module.css

```diff
@@ -53,22 +53,22 @@
 }
 
 .add {
-  margin-right: 16px;
+  margin-right: 12px;
```

Comment:
are we using spacing vars in the css files now or only in emotion?

Reply:
we can for sure just lazy here 😢

Reply:
lol ya np at all. More asking for my own sake

Reply:
updated so you can see :)